crates/biome_formatter/Cargo.toml

```diff
@@ -15,6 +15,7 @@ version              = "0.2.0"
 
 [dependencies]
 biome_console     = { workspace = true }
+biome_deserialize = { workspace = true }
```

Comment:
Where do we use this crate?

Reply:
It is used to implement LineWidth deserializer

Reply:
Ah shoot sorry, I read `biome_console` 🤣

Reply:
Oh maybe you though about [biome_js_formatter's Crago.toml](https://github.com/biomejs/biome/pull/683/files#diff-02ab6043dc93f21e4b8e944e3657bad672e6303e7ac00fd67a5bb0975ad66317)?